PersonApi/V1/Boundary/Request/PersonRequestObject.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using PersonApi.V1.Domain;
+using PersonApi.V1.Infrastructure;
+
+namespace PersonApi.V1.Boundary.Request
+{
+    public class PersonRequestObject
+    {
+        public Guid Id { get; set; }
```

Comment:
I don't think you need to Id in the request object

Reply:
It would work for future reference when we wish to update a person :) it also allows to transform ID to DB entity as per the change the double Michaels requested upstairs !